swim/state_transitions_test.go

```diff
@@ -109,12 +113,42 @@ func (s *StateTransitionsSuite) TestSuspectBecomesFaulty() {
 	s.Require().NotNil(member, "expected cannot be nil")
 
 	s.stateTransitions.ScheduleSuspectToFaulty(*member)
-	s.NotNil(s.stateTransitions.timer(member.Address), "expected suspicion timer to be set")
+	s.NotNil(s.stateTransitions.timer(member.Address), "expected state transtition timer to be set")
 
-	time.Sleep(5 * time.Millisecond)
+	s.clock.Add(5 * time.Millisecond)
 	s.Equal(Faulty, member.Status, "expected member to be faulty")
 }
 
+func (s *StateTransitionsSuite) TestFaultyBecomesTombstone() {
+	s.stateTransitions.timeouts.Faulty = time.Millisecond
```

Comment:
Do we need small timeouts in the unit tests, also when we mock? Can we use the defaults?